store/filecache.go

```diff
@@ -1,7 +1,6 @@
 package store
```

Comment:
replace all use of filecache with `FileStore` in `trustmanager`? Provides the exact same functionality.